include/libultraship/libultra/abi.h

```diff
@@ -446,13 +447,20 @@ typedef short ENVMIX_STATE[40];
         _a->words.w1 = (u32)(addr);                                                                 \
     }
 
-#define aDuplicate(pkt, count, dmemi, dmemo, a4)                                                      \
-    {                                                                                                 \
-        Acmd* _a = (Acmd*)pkt;                                                                        \
-                                                                                                      \
-        _a->words.w0 = (_SHIFTL(A_DUPLICATE, 24, 8) | _SHIFTL(count, 16, 8) | _SHIFTL(dmemi, 0, 16)); \
-        _a->words.w1 = _SHIFTL(dmemo, 16, 16) | _SHIFTL(a4, 0, 16);                                   \
+#define aDuplicate(pkt, numCopies, dmemSrc, dmemDest)                                                       \
+    {                                                                                                       \
+        Acmd* _a = (Acmd*)pkt;                                                                              \
+                                                                                                            \
+        _a->words.w0 = (_SHIFTL(A_DUPLICATE, 24, 8) | _SHIFTL(numCopies, 16, 8) | _SHIFTL(dmemSrc, 0, 16)); \
+        _a->words.w1 = _SHIFTL(dmemDest, 16, 16) | _SHIFTL(0x80, 0, 16);                                    \
     }
+//#define aDuplicate(pkt, count, dmemi, dmemo, a4)                                                      \
+//    {                                                                                                 \
+//        Acmd* _a = (Acmd*)pkt;                                                                        \
+//                                                                                                      \
+//        _a->words.w0 = (_SHIFTL(A_DUPLICATE, 24, 8) | _SHIFTL(count, 16, 8) | _SHIFTL(dmemi, 0, 16)); \
+//        _a->words.w1 = _SHIFTL(dmemo, 16, 16) | _SHIFTL(a4, 0, 16);                                   \
+//    }
```

Comment:
it's not clear why this is commented out instead of removed. if the idea is to keep it around as a reference it'd be nice to add a comment explaining that, otherwise just rip it out